executor_compromise_the_return.md

```diff
@@ -129,11 +134,11 @@ auto sender2 = ex.make_value_task(sender1, fn);
 sender2.submit(p2);
 ```
 
-...where `p2` is possibly a promise corresponding to `fut2` from above, though it need not be.
+... where `p2` is possibly a promise corresponding to `fut2` from above, though it need not be.
 
 [P0443] `execute` functions return a future. The type of the future is under the executor's control. By splitting `execute` into lazy task construction and a (`void`-returning) work submission API, we enable lazy futures because the code returning the future can rely on the fact that submit will be called by the caller. With that knowledge, the lazy future is safe to return because we can rely on it being run.
 
-We optionally lose the ability to block on completion of the task at task construction time. As submit is to be called anyway (except for the pure oneway executor case where submit is implicit) it is cleaner to apply the blocking semantic at this point if we are to have it at all. In particular, this approach allows us to build executors that return senders that block on completion but are still lazy.
+We optionally lose the ability to block on completion of the task at task construction time. As `submit` is to be called anyway (except for the pure oneway executor case where submit is implicit) it is cleaner to apply the blocking semantic to `submit` point. In particular, this approach allows us to build executors that return senders that block on completion but are still lazy.
```

Comment:
"to `submit` point" reads awkwardly. Did you forget a word or two?

Reply:
Sorry, this got mangled.